framework/validators/BooleanValidator.php

```
@@ -53,8 +53,11 @@ public function init()
             $this->message = Yii::t('yii', '{attribute} must be either "{true}" or "{false}".');
         }
 
-        if (Yii::$app?->useJquery && !$this->clientScript instanceof ClientValidatorScriptInterface) {
-            $this->clientScript ??= ['class' => BooleanValidatorJqueryClientScript::class];
+        if ($this->clientScript === null && (Yii::$app->useJquery ?? false)) {
+            $this->clientScript = ['class' => BooleanValidatorJqueryClientScript::class];
+        }
+
+        if ($this->clientScript !== null && !$this->clientScript instanceof ClientValidatorScriptInterface) {
             $this->clientScript = Yii::createObject($this->clientScript);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Yii' in method 'init'. (undefined)

(StaticAccess)